readme.md

```diff
@@ -122,6 +121,7 @@ Although this [must be run manually](#linting-your-changes), developers may wish
 
 The following dependencies aren't needed by most people, and are not included in Git submodules:
 
+* To generate developer documentation: [Sphinx](http://sphinx-doc.org/), version 3.4.1
```

Comment:
This is inconsistent with the requirements file. I can imagine that someone reports an issue stating that readme asks to install specific version of Sphinx whereas requirements file installs whatever is the latest version at the time of installation. It'd be great to either remove specifif version number from the readme, or enforce version 3.4.1 in the requirements file. I personally prefer the latter as we cannot guarantee that dev docs would always be buildable with the most recent version of Sphinx.